bitwarden_license/test/Commercial.Core.Test/AdminConsole/ProviderFeatures/RemoveOrganizationFromProviderCommandTests.cs

```diff
@@ -85,161 +86,154 @@ public class RemoveOrganizationFromProviderCommandTests
     }
 
     [Theory, BitAutoData]
-    public async Task RemoveOrganizationFromProvider_NoStripeObjects_MakesCorrectInvocations(
+    public async Task RemoveOrganizationFromProvider_OrganizationNotStripeEnabled_MakesCorrectInvocations(
         Provider provider,
         ProviderOrganization providerOrganization,
         Organization organization,
         SutProvider<RemoveOrganizationFromProviderCommand> sutProvider)
     {
-        organization.GatewayCustomerId = null;
-        organization.GatewaySubscriptionId = null;
-
         providerOrganization.ProviderId = provider.Id;
 
-        var organizationRepository = sutProvider.GetDependency<IOrganizationRepository>();
+        organization.GatewayCustomerId = null;
+        organization.GatewaySubscriptionId = null;
 
         sutProvider.GetDependency<IOrganizationService>().HasConfirmedOwnersExceptAsync(
                 providerOrganization.OrganizationId,
-                Array.Empty<Guid>(),
+                [],
                 includeProvider: false)
             .Returns(true);
 
-        var organizationOwnerEmails = new List<string> { "a@gmail.com", "b@gmail.com" };
+        var organizationRepository = sutProvider.GetDependency<IOrganizationRepository>();
 
-        organizationRepository.GetOwnerEmailAddressesById(organization.Id).Returns(organizationOwnerEmails);
+        organizationRepository.GetOwnerEmailAddressesById(organization.Id).Returns([
+            "a@b.com",
```

Comment:
Nit: I prefer to use the domain `example.com` that is reserved for testing purposes